enhancements/windows-containers/health-management.md

```diff
@@ -125,6 +126,11 @@ N/A
 * WMCO will not default to containerd as the container runtime until the 6.0.0 timeframe. Therefore, we will be using a 
   runtime flag `dockerRuntime=false` introduced by the containerd enhancement to develop during the 5.y.z cycle.
   This flag will be dropped in WMCO 6.0.0.
+* Although WICD relies on the services ConfigMap data for all service configuration information, it will not be 
+  authorized to get the resource as `system:node` kubeconfig permissions do not blanket allow GET or LIST requests for
+  ConfigMaps. The resources being pulled must be related to the Node. To mitigate this, we plan to run WICD in a host
+  process container (scratch image without any Windows OS bits) on each node through a DaemonSet, attaching the WMCO
```

Comment:
Alternatively, can't you create a `kubeconfig` for the SA and then pass that to WICD and not have to use a host container?

Reply:
To be clear, it is preferable to use host process containers but I want to make sure that we are not going to run into issues with Brew / CPaaS.

Reply:
I dont think this portion of the enhancement should be added at the moment. We have a clear solution for the permission issues when bootstrapping, but I am not fully certain yet of how we should handle the controller permissions.

To answer @aravindhp's questions:

>I thought a scratch image was [not possible](https://kubernetes.slack.com/archives/C0SJ4AFB7/p1652884354539439) with Windows containers. @sebsoto have you worked around the issue?

I've been able to build and run WICD as a host process container, built from scratch with docker buildx.
The concept works. In execution things are not ready.
1) Buildah, which is what is used in our build pipeline, is unable to build Windows containers. I've opened [an issue](https://github.com/containers/buildah/issues/4010) for this, and hope to eventually resolve it
2) Microsoft has not made the requirements for a scratch image public. Even though I've gotten it to work, theres no guarantee that the requirements will not change in the future, breaking compatibility.

> Alternatively, can't you create a kubeconfig for the SA and then pass that to WICD and not have to use a host container?

For the short term I think this might be the required solution. Before that there is some research that needs to be done with regards to SA token rotation. And yes a new Service account with only the required permissions should be used.

Reply:
Sure, I'll drop the no -bootstrap discussion.